app/src/components/card/searchCard.tsx

```diff
@@ -38,60 +40,92 @@ const contentTypeTag = (result: SearchCardType) => {
       ? result.contentType === "Image" && result.containsMultipleCaptures
         ? "Multiple images"
         : result.contentType
-      : result.recordType;
+        ? result.contentType
+        : null
+      : result.recordType
+      ? result.recordType
+      : null;
```

Comment:
`contentTypeTag` is not easy to read. Can you break it down into smaller variables?